gstd/gstd_pipeline.c

```diff
@@ -375,6 +398,14 @@ gstd_pipeline_set_property (GObject * object,
       }
       self->state = g_value_get_object (value);
       break;
+    case PROP_GRAPH:
```

Comment:
If the property is not writable this is not needed (and hence the graph variable at all)

Reply:
Yes I will remove that, but I need the graph variable to store the text returned by `gst_debug_bin_to_dot_data`.